_faq/resource/enforcers.md

```diff
@@ -16,3 +16,9 @@ For details about each of the enforcers, see the
 | --------------------------------------- |
 | Firewalls |
 
+{: .table .table-striped}
```

Comment:
This section lists the Google Cloud Platform (GCP) resources that currently have coverage using the Real-Time Enforcer. If a resource you're interested in isn't listed, please open an issue or contribute!

For details about each of the resources, see the
[Real-Time Enforcer]({% link _docs/latest/configure/real-time-enforcer/default-policies.md %}) guide.